internal/repoupdater/client.go

```diff
@@ -128,20 +127,15 @@ func (c *Client) RepoLookup(
 		return MockRepoLookup(args)
 	}
 
-	span, ctx := ot.StartSpanFromContext(ctx, "Client.RepoLookup") //nolint:staticcheck // OT is deprecated
+	tr, ctx := trace.New(ctx, "repoupdater", "RepoLookup",
+		attribute.String("repo", string(args.Repo)))
 	defer func() {
 		if result != nil {
-			span.SetTag("found", result.Repo != nil)
+			tr.SetAttributes(attribute.Bool("found", result.Repo != nil))
 		}
-		if err != nil {
-			ext.Error.Set(span, true)
-			span.SetTag("err", err.Error())
-		}
-		span.Finish()
+		tr.SetError(err)
+		tr.Finish()
```

Comment:
in this case would you be able to use tr. FinishWithErr ?
```suggestion
		tr.FinishWithErr(&err)
```

Reply:
Yep, sure enough